src/gui/DatabaseWidget.cpp

```diff
@@ -81,12 +81,12 @@ DatabaseWidget::DatabaseWidget(Database* db, QWidget* parent)
     mainLayout->addWidget(m_messageWidget);
     mainLayout->addLayout(layout);
     m_mainSplitter = new QSplitter(m_mainWidget);
-    m_mainSplitter->setChildrenCollapsible(false);
     m_detailSplitter = new QSplitter(m_mainWidget);
     m_detailSplitter->setOrientation(Qt::Vertical);
     m_detailSplitter->setChildrenCollapsible(true);
 
     QWidget* rightHandSideWidget = new QWidget(m_mainSplitter);
+    m_mainSplitter->setCollapsible(0, false);
```

Comment:
I would prefer you do it the other way round: leave childrenCollapsible at false and set collapsible to true for child No. 1.

However, I'm generally not a fan of collapsible panels. They tend to disappear with hardly any to no indication where they went. I think this needs some visual indicator telling the user how to get their panel back. What do you think @droidmonkey @TheZ3ro ?

Reply:
Should be same as the details view.

Reply:
Which is simply collapsible, too, and I don't like it

Reply:
I think we should have some visual indicator, but so this won't be in 2.3.0

Reply:
> I would prefer you do it the other way round: leave childrenCollapsible at false and set collapsible to true for child No. 1.
Default value for childrenCollapsible is true. I used this way because of simplicity.

Reply:
The default was explicitly set to false and you changed it back to true. I would prefer false as the default. Right now we only have two child widgets, but that may not always be the case.